src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -62,5 +152,34 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
             border.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
             return border;
         }
+
+        /// <summary>
+        /// Determines if the border is being moused over and shows the info accordingly
+        /// </summary>
+        private void Border_ToolTipOpening(object sender, ToolTipEventArgs e)
+        {
+            var border = (Border)sender;
+            e.Handled = true;
+
+            bool KeepOpen()
+            {
+                var mousePoint = Mouse.GetPosition(border);
+                return !(mousePoint.X > border.ActualWidth || mousePoint.X < 0 || mousePoint.Y > border.ActualHeight || mousePoint.Y < 0);
+            }
+
+            var toolTipPresenter = _toolTipService.CreatePresenter(_textView, new ToolTipParameters(trackMouse: true, ignoreBufferChange: false, KeepOpen));
+            var startToolTip = StartToolTipServiceAsync(toolTipPresenter);
+        }
+
+        /// <summary>
+        /// Waits for the description to be created and updates the tooltip with the associated information
+        /// </summary>
+        private async Task StartToolTipServiceAsync(IToolTipPresenter toolTipPresenter)
+        {
+            var uiList = await await Task.Run(() => CreateDescriptionAsync(CancellationToken.None).ConfigureAwait(false)).ConfigureAwait(false);
```

Comment:
Pass _threadingContext.DisposalToken as the cancellation token for CreateDescriptionAsync? If you do there's also a cancellationToken parameter to Task.Run() to call as well.

Reply:
Fixed